meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -81,7 +81,11 @@ impl SegmentAnalytics {
         let user_id = user_id.unwrap_or_else(|| Uuid::new_v4().to_string());
         write_user_id(&opt.db_path, &user_id);
 
-        let client = HttpClient::default();
+        let client = reqwest::Client::builder()
+            .connect_timeout(Duration::new(10, 0))
+            .build()
+            .unwrap();
```

Comment:
are you sure this can't fail?
https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.build